scripts/linux/install-kubernetes.sh

```diff
@@ -13,16 +13,40 @@ while true; do
   esac
 done
 
+ansible_debug="$1"
+
+echo "Ansible debug flag: $ansible_debug"
+
+verbose_flag=
+if [ "$ansible_debug" = "on" ]; then
+    verbose_flag="-vv"
+fi
+
+echo "Verbose verbose_flag value: $verbose_flag"
+
 echo "Ensure the Docker service is enabled and running"
+
 systemctl enable docker
 systemctl restart docker
 
 inventory="/etc/"$inventory
 
 echo "Running Ansible playbooks against $inventory inventory"
-docker run --rm \
-    -v /vagrant/ansible:/etc/ansible \
-    -v /vagrant/ansible/playbooks/files/tls:/opt/tls/self_signed \
-    --net=host \
-    ferrarimarco/open-development-environment-ansible:2.7.12-alpine \
-    /bin/sh -c "ansible-galaxy install -r /etc/ansible/requirements.yml && ansible-playbook -i $inventory /etc/ansible/playbooks/kubernetes.yml && ansible-playbook -i $inventory /etc/ansible/playbooks/openssl-self-signed-certificate.yml"
+
+if [ "$ansible_debug" = 'on' ]; then
```

Comment:
To avoid repetitions, we could put this command in a function

Reply:
of course I agree, but I'm not a big expert in shell programming ;-) 
1) will it work to use the function and then redirect the output in one of the two cases?
2) inside the function is there visibility of the verbose_flag variable? (I guess so...)

Reply:
we have avoided the need of the "if" because we always redirect to a file